### ISSUE TITLE
Platform specific code

### ISSUE THREAD

**Issue by clayrisser** [Place1/react-native-gtk#24]
Would it be possible to support `*.gtk.js` files while ignoring `*.ios.js` and `*.android.js`.

https://facebook.github.io/react-native/docs/platform-specific-code.html

**Comment by ewnd9** [comment]
Would be interesting to research other desktop implementations:

- https://github.com/Microsoft/react-native-windows
- https://github.com/ptmt/react-native-macos
- https://github.com/status-im/react-native-desktop (driven by Qt, forked from Canonical)

**Comment by Place1** [comment]
yep. @jamrizzi we do want to support as much of the react-native APIs as we can.

Given this package is in its infancy my focus is on the GTK rendering, layout and styling as those will give us a good idea of how feasible this project is. I don't see this particular feature (platform api) as a priority; but yes, we do want this :D
